voxblox_ros/include/voxblox_ros/mesh_vis.h

```diff
@@ -83,8 +97,8 @@ inline void heightColorFromVertex(const Point& vertex,
 }
 
 inline void fillMarkerWithMesh(const MeshLayer::ConstPtr& mesh_layer,
```

Comment:
The reason I say that is that if any client of your function is has a raw pointer to something (rather than a smart pointer) it makes the function unusable, unless you then construct a smart pointer with an empty destructor (hack city). Or maybe I'm wrong and there's something I'm missing.